apps/charterafrica/package.json

```diff
@@ -40,6 +40,7 @@
     "@emotion/react": "^11.10.5",
     "@emotion/server": "^11.10.0",
     "@emotion/styled": "^11.10.5",
+    "@mui/icons-material": "^5.10.9",
```

Comment:
```suggestion
```

We don't use mui icons, download relevant icons from Figma (as SVG) and use them.